### DISCUSSION TITLE
突然一个图的caption在图的上方

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#122] [Q&A]
这种现象是正常的吗？
![Screenshot 2022-04-14 184149](https://user-images.githubusercontent.com/45897471/163373345-3d1d0d02-b54d-4d98-b342-291c44882264.jpg)

> \begin{figure}
      \centering
      \caption[EX}
      \label{fig:EX}
      \includegraphics[width=\textwidth]{EX.png}
    \end{figure}

**Comment by tanukihee** [comment]
？你要不要好好读一读你写的代码？

**Reply by earthmoon** [reply]
> ？你要不要好好读一读你写的代码？

代码这么讲究的啊，我以为只是提取信息...

**Reply by tanukihee** [reply]
算了，只能告诉你，`\caption` 写在哪就在哪儿

代码很正常，现象也很正常，不正常的地方，就请你自己想了

**Reply by earthmoon** [reply]
好的吧
